shoppingList/css/style.css

```
@@ -0,0 +1,223 @@
+/*reset css*/
+@import "/shoppingList/css/reset.css";
+
+html,body {	margin: 0;	padding: 0;
+        background-color: rgb(235, 235, 235);
+        font-family: "Gaegu", sans-serif;
+        font-weight: 400;
+        font-style: normal;
+}
+
+
+/** header **/
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    position: fixed;
+    width: 100%;
+    height: 10%;
+    background-color: rgb(217, 217, 217);
+    z-index: 5;
+
+}
+
+#header_logo {
+    padding: 3rem;
+    font-size: 40px;
+}
+
+.header_menu {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+}
+
+.header_top {
+    display: flex;
+    flex-direction: column;
+    
+}
+
+
+/**banner slide**/
+.slider {
+    overflow: hidden;
+    width: 1100px;
+    height: 300px;
+    box-sizing: border-box;
+    align-self: flex-end;
+    margin-top: 120px;
+    margin-right: 200px;
+
+}
+
+.image-box {
+    width: 4000px;
+    height: 100%;
+    display: flex;
+    flex-wrap: nowrap;
+    animation: bannermove 20s linear infinite;
+}
+
+.image-box > div{
+    margin: 0px -1px 0px 0px;
+}
+
+@keyframes bannermove {
+    0% {
+        transform: translate(0, 0);
+    }
+
+    100% {
+        transform: translate(-50%, 0);
+    }
+}
+
+/**nav**/
+nav {
+    position: fixed;  
+    margin-top: 15rem;
+    background-color: white;
+    display: flex;
+    margin-left: 2rem;
+    border-radius: 3%;
+    padding-top: 5rem;
+    padding-bottom: 5rem;
+    
+
+}
+
+
+li {
+    padding: 0.5rem;
+    font-size: 20px;
+    margin:0 3rem;
+}
+
+
+a {
+    text-decoration-line: none;
+    text-decoration-color: none;
+    color: black;
+}
+
+
+li:hover {
+    background-color: rgb(217, 217, 217);
+    
+}
+
+
+a:hover {
+    color: white;
+}
+
+
+/**section**/
+section {
+    scroll-margin-top: 6rem;
+}
+
+main {
+    padding: 0rem 15rem;
+}
+
+.all {
+    display: grid;
+    grid-template-rows:1fr 1fr;
+    grid-template-columns: repeat(auto-fit, minmax(170px, 1fr));
+    grid-gap:2rem;
+    justify-content: space-evenly;
+    margin-bottom: 5rem;
+    padding: 0px;
+}
+img {
+    width: 100%;
+    height: 80%;
+    object-fit: cover;
+}
+
+p {
+    margin: 0.5rem;
+}
+
+.top_wrapper {
+    display: grid;
+    grid-template-rows: 1fr;
+    grid-template-columns: repeat(auto-fit, minmax(170px, 1fr));  
+    grid-gap:2rem;
+    justify-content: space-evenly;
+    margin-bottom: 5rem;
+}
+
+.pants_wrapper {
+    display: grid;
+    grid-template-rows: 1fr;
+    grid-template-columns: repeat(auto-fit, minmax(170px, 1fr)); 
+    grid-gap:2rem;
+    justify-content: space-evenly;
+    margin-bottom: 5rem;
+}
+
+.outer_wrapper {
+    display: grid;
+    grid-template-rows: 1fr;
+    grid-template-columns: repeat(auto-fit, minmax(170px, 1fr));
+    grid-gap:2rem;
+    justify-content: space-evenly;
+}
+
+
+.outer, .top, .pants {
```

이건 클래스에서 똑같은 스타일을 사용하니까 클래스명을 하나로 통일하는 것도 좋을 것 같아! 
그럼 html 코드에서도 다 다른 클래스가 아니라 똑같은 클래스를 사용할 수 있어서 좋을 것 같아 ~

```suggestion
.item {
```

---

그러게....다른 스타일을 쓸 줄 알고 나눴나봐 통일할걸!!!!!!